calico-cloud/image-assurance/understanding-scan-results.mdx

```diff
@@ -11,7 +11,32 @@ In the left navbar in Manager UI, click **Image Assurance**, **All Scanned Image
 ## All Scanned Images tab
 
 This tab lists scanned images if you have enabled or used one of the [Image Assurance scanners](./scanners/overview).
-To manage your scan results, you can filter the list results or delete particular scan result items.
+
+#### Filter Scan Results
+
+You can use different combinations of filters to refine your results, making it easier to monitor and focus on relevant vulnerabilities for your remediation efforts.
+
+#### Delete Scan Results
+
+If a scan result is no longer relevant to your remediation effort, you can delete it by selecting multiple results and choosing the **Delete** option from the Bulk Actions dropdown.
+
+:::note
+
+**Deleted Results**: Deleting a scan result removes it from the **All Scanned Images** tab. However, if the image is currently running, its record will still appear in the **Running Images** tab, with the number of instances tracked.
+
+:::
```

Comment:
Do we really need this? I'm not sure we'd expect deleting a result to mean ALL results are removed for that image.

Reply:
Agree. This can be omitted as users likely expect this behavior.